independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/build.yml

```diff
@@ -48,4 +52,4 @@ jobs:
         run: mvn -B clean verify -Dno-format
 
       - name: Build with Maven (Native)
-        run: mvn -B verify -Dnative -Dquarkus.native.container-build -Dnative.surefire.skip
+        run: mvn -B verify -Dnative "-Dquarkus.native.container-build" "-Dnative.surefire.skip"
```

Comment:
This is needed when building in Windows

Reply:
I wonder if specifying `shell: bash` would be less brittle? Not exactly sure it will fix things but it might be more future proof if it does.
If it does, let's generalize it in this file.

Reply:
@gsmet good idea, just tested and it works fine